hapistrano.cabal

```diff
@@ -35,22 +35,22 @@ library
   exposed-modules:     System.Hapistrano
   other-modules:       System.Hapistrano.Types
   build-depends:       base >= 4.5 && < 4.10
-                     , either
-                     , filepath
-                     , mtl
-                     , process
-                     , time-locale-compat
-                     , time
-                     , transformers
+                     , either >= 4.0 && < 4.6
+                     , filepath >= 1.3 && < 1.5
+                     , mtl >= 2.0 && < 3.0
+                     , process >= 1.2 && < 1.5
+                     , time-locale-compat >= 0.1 && < 1.0
+                     , time >= 1.4.1 && < 2.0
+                     , transformers > 0.3 && < 1.0
```

Comment:
We don't need version constraints for boot libraries (filepath, process, time, and transformers). I don't think we need a particular version for any of those, so the version constraints for base should be enough. The ones for either, mtl, and time-locale-compat are fine.